specifications/sep-payment-channel-transactions.md

```diff
@@ -229,24 +229,25 @@ multisig accounts. F has source account E, and sequence number set to s.
   The `CHANGE_TRUST` operations configure the trustlines with the maximum limit,
   which is the maximum value of an `int64`, `0x7FFFFFFFFFFFFFFF`.
 
-- C_i, see [Update](#Update) process.
+- C_i, see [Payment](#Payment) process. The setup C_i will be a Payment of 0.
```

Comment:
imo this is clearer now. However I'm interested if you think this actually is less clear

Reply:
Hmm, technically there won't be a payment. We'll omit the payment operation if there is no payment to make.

Reply:
```suggestion
- C_i, with no payment, see [Payment](#Payment) process.
```

Reply:
I realize looking at the text further down this isn't particularly obvious, so I made it clearer in https://github.com/stellar/experimental-payment-channels/pull/167.